extensions/timeseries/src/main/java/ai/djl/timeseries/timefeature/TimeOffset.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package ai.djl.timeseries.timefeature;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/** This is a class use to get multiple and granularity from frequency string. */
+public class TimeOffset {
```

Comment:
How are users going to use this class? 

It looks like you're trying to use it as a utility to parse a certain string format, but it's not an accepted type at the moment for any method.